workbench-core/example/infrastructure/integration-tests/tests/users/createUser.test.ts

```diff
@@ -43,6 +43,9 @@ describe('userManagement create user integration test', () => {
   });
 
   it('should return a 400 error when a user with the provided email already exists', async () => {
+    // create user
+    await adminSession.resources.users.create(user);
```

Comment:
Do we remove this user eventually?

Reply:
Yep! Users get cleaned up in the `afterAll` block